src/config/firebaseRoar.js

```diff
@@ -8,8 +8,15 @@ const isEmulated = import.meta.env.VITE_FIREBASE_EMULATOR === 'true';
 const useSandbox = import.meta.env.VITE_FIREBASE_DATA_SOURCE === 'sandbox';
 const isStaging = import.meta.env.VITE_STAGING_BUILD === 'true';
 
+function setDebugToken(config) {
+  const debugToken = import.meta.env.VITE_APPCHECK_DEBUG_TOKEN;
```

Comment:
Do we want to apply [this suggestion](https://github.com/yeatmanlab/roar-dashboard/pull/743/files#r1723581902) here to allow engineers to generate their own debug token instead of passing a single token around? 
```suggestion
  const debugToken = import.meta.env.VITE_APPCHECK_DEBUG_TOKEN || (import.meta.env.VITE_APPCHECK_DEBUG_TOKEN === 'true')

```

Reply:
Added.

Reply:
After some testing, this line seems to cause our Cypress tests to fail. I think that it is generating a new debug token each time, and that token won't be valid until it is added to the Firestore app check console. I am going to leave this line out for now and investigate further.

Reply:
Odd – when I tried it, it was only generating new tokens when `VITE_APPCHECK_DEBUG_TOKEN` was set to `true`